### PR TITLE
Release notes for 1.4.0-preview2 and 0.16.0-preview2.

### PR DESCRIPTION
**Pull request by codemzs**
We have released 1.4.0-preview2 and 0.16.0-preview2 versions of our nugets on nuget.org. These are the release notes.

### CODE REVIEW

File: docs/release-notes/1.4.0-preview2/release-1.4.0-preview2.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- Fix memory leak in TensorflowTransformer ([#4223](https://github.com/dotnet/machinelearning/pull/4223))
+- Enable permutation feature importance to be used with model loaded from disk ([#4262](https://github.com/dotnet/machinelearning/pull/4262))
+- `IsSavedModel` returns true when loaded TensorFlow model is a frozen model ([#4262](https://github.com/dotnet/machinelearning/pull/4197))
+- Exception when using `OnnxSequenceType` attribute directly without specify sequence type ([#4272](https://github.com/dotnet/machinelearning/pull/4272)
+
+## **Samples**
+- TensorFlow full model retrain sample ([#4127](https://github.com/dotnet/machinelearning/pull/4127))
+
+## **Breaking Changes**
+None.
```
[the remark this answers is not in the document]
Comment (luisquintanilla) [reply docs/release-notes/1.4.0-preview2/release-1.4.0-preview2.md]: For breaking changes, I'd note that in order to use file paths, you now have to also use the `LoadImages` transform to get the string image path into `byte[]` or ImageType.  Previously, you only needed to provide a string path.

[the remark this answers is not in the document]
Comment (CESARDELATORRE) [reply docs/release-notes/1.4.0-preview2/release-1.4.0-preview2.md]: You need to use byte[] if you want to be able to score with in-memory images.

The API should be able to work without in-memory images, too, but I got some issues with that yesterday, too. We have to isolate that case, but I don't think it is an scenario we want to promote, though.

Working with byte[] you get the best of both worlds, I think. You can load image files then score with in-memory images. I think it is more flexible. 

- Using `string ImageFilePath` will only allow you to work with file paths when scoring.
- Using byte[] Image allows you to also use file paths before converting to byte[] or getting the in-memory image from a different channel (HTTP, streaming, etc.).

[the remark this answers is not in the document]
Comment (codemzs) [reply docs/release-notes/1.4.0-preview2/release-1.4.0-preview2.md]: @luisquintanilla This is not a breaking API change as this is an in preview API.

[the remark this answers is not in the document]
Comment (codemzs) [reply docs/release-notes/1.4.0-preview2/release-1.4.0-preview2.md]: That breaking change section is specifically for GA to GA release. In preview means anything can change anytime without notice. I expect people to follow samples to know how to use API.

**Pull request merged**